x-pack/plugins/security/server/routes/authorization/roles/get_all_by_space.test.ts

```diff
@@ -320,5 +320,72 @@ describe('GET all roles by space id', () => {
         ],
       },
     });
+
+    getRolesTest(`filters roles with reserved only privileges`, {
+      apiResponse: () => ({
```

Comment:
**question:** Should we also include a third role in the test that doesn't have `metadata._reserved` and also lacks Kibana `base` and `feature` privileges? This seems to be a distinct case in the code. Or how is this API supposed to behave for non-reserved roles without Kibana privileges?

Reply:
Added in 19c1346f3d70a1f5d11fc4126484af56b68b4107